app/facades/attendance_show_facade.rb

```diff
@@ -21,7 +21,7 @@ def unassigned_zoom_aliases
   end
 
   def student_attendances
-    @attendance.student_attendances.includes(:student).by_attendance_status
+    @attendance.student_attendances.includes(:student).where.not(students: { name: "Instructor" }).by_attendance_status
```

Comment:
Good solution, Let's create some testing for it to ensure it works exactly as intended.

Reply:
Thank you, and yes great call. We can add some facade tests as well as feature tests.